### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by elizabethcan**
Some front end tests, most of which are in `Container` since that's where most of my functions are.

### CODE REVIEW

File: client/src/__tests__/container.test.js [hunk 1]
```diff
@@ -12,9 +13,105 @@ it('should render correctly with no props', () => {
...
 
-  expect(getReviews).toHaveBeenCalledTimes(1);
-});
\ No newline at end of file
+  expect(getData).toHaveBeenCalledTimes(1);
+});
+
+// it('should display the modal when showModal state is true', () =>{
+//   const container = mount(<Container />);
+//   container.setState({modalReview: 0, showModal: true});
```
Comment (JonBeyond) [inline client/src/__tests__/container.test.js]: oh, is this how you set state variables in a wrapper? it makes a lot of sense.

File: client/src/__tests__/container.test.js [hunk 2]
```diff
@@ -12,9 +13,105 @@ it('should render correctly with no props', () => {
...
+//   console.log(container.find('.averageStars').debug());
+
+//   expect(container.find('.averageStars')).toHaveLength(3);
+// });
+
+it('should set the ReviewContainer limit to 20 when showMore is clicked', () => {
+  const showMore = jest.spyOn(Container.prototype, 'showMore');
+  const container = shallow(<Container />);
+  container.instance().showMore();
+  container.update();
```
Comment (JonBeyond) [inline client/src/__tests__/container.test.js:27]: Likewise, I didn't know you could do these commands here.

File: client/src/components/Container.jsx [hunk 3]
```diff
@@ -112,10 +112,10 @@ class Container extends React.Component {
     return (
       <div style={containerStyles.container}>
         <div style={{display: 'flex'}}>
-          <h2 style={containerStyles.header}>
+          <div style={containerStyles.header}>
```
Comment (JonBeyond) [inline client/src/components/Container.jsx:119]: I didnt notice that you had `h2` in here before, good catch

**Review by JonBeyond — APPROVED** [review]
overall looks good; what % test coverage do you have of your 1) functions and 2) 'total lines'?

File: client/src/__tests__/container.test.js (hunk 2 above)
Comment (elizabethcan) [reply client/src/__tests__/container.test.js:27]: `.instance().methodName()` is how you invoke the method in the test. `.update()` will update the component if state changed

File: client/src/components/Container.jsx (hunk 3 above)
Comment (elizabethcan) [reply client/src/components/Container.jsx:119]: it looks a lot better as a div

**Comment by elizabethcan** [comment]
I have to say, I'm really having a hard time understanding how to write these tests. I removed the ones that are failing and I'm going to merge the ones that are passing right now just to have them in master so that I can fix them now with my branches, and I'll create a new branch right after I merge this to continue trying to hammer out some more tests. I'm only at about 30% function coverage and 40% line coverage in my Container component

**Pull request merged**